packages/meditrak-server/src/routes/getSocialFeed.js

```diff
@@ -94,8 +94,7 @@ const getLeaderboardFeedItem = async models => {
     creation_date: new Date(),
     template_variables: {
       title: 'Tupaia Leaderboard',
-      // Enable this to display pigs in leaderboard once pig ledger is accurate.
-      hasPigs: false,
+      hasPigs: false, // this can be set to true if the meditrak-app table is fixed
```

Comment:
Pig ledger is now correct, but I tried turning it on and the table in meditrak overflows weirdly. I don't imagine we'll ever fix it, so could probably just delete all the related code, but have already deleted a lot in this PR and that would involve a meditrak release so have left for another day (and perhaps we will fix it up and make it look pretty)

Reply:
Is there a ticket for that?

Reply:
No, because we'll never do it